presto-spi/src/main/java/com/facebook/presto/spi/SchemaUtil.java

```diff
@@ -25,7 +27,7 @@ static String checkNotEmpty(String value, String name)
             throw new NullPointerException(name + " is null");
```

Comment:
Is there a reason we wouldn't fix this one also? Or is it not possible?

Reply:
There is no reason and it's possible:)
I'll update shortly.